packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.mdx

```diff
@@ -66,4 +66,41 @@ When [displaying products as a table](https://docs.coveo.com/en/atomic/latest/us
   </atomic-product-template>
 </atomic-commerce-product-list>
 ```
+
+### Displaying a specific product listing
+If you want `atomic-commerce-product-list` to display a specific product listing, you will need to ensure that the `atomic-commerce-interface` at
+level component of the hierarchy for this interface has its type set to `product-listing`.
+
+You will also need to ensure that configuration being used to initialize the interface has the `view` url of the configuration
+[`context`](https://docs.coveo.com/en/headless/latest/reference/interfaces/Commerce.ContextOptions.html) set to a configured listing page.
+
+
+For example,
+```html
+<script type="module">
+  (async () => {
+    const {
+      getSampleSearchEngineConfiguration
+    } = await import('https://static.cloud.coveo.com/headless/v3/headless.esm.js');
+    const {context, ...restOfConfiguration} = getSampleCommerceEngineConfiguration();
+
+    const configuration = {
+      context: {
+        ...context,
+        view: {
+          url: `${context.view.url}/browse/promotions/ui-kit-testing`,
+        },
+      },
+      ...restOfConfiguration,
+    };
+
+    await customElements.whenDefined('atomic-commerce-interface');
+    const commerceInterface = document.querySelector('atomic-commerce-interface');
+    await commerceInterface.initialize(configuration);
+    await commerceInterface.executeFirstRequest();
+  })();
+</script>
```

Comment:
There's this ol' nagging practice of using async IIFE, but we don't need those anymore.

```suggestion
<script type="module">
  const {
    getSampleSearchEngineConfiguration
  } = await import('https://static.cloud.coveo.com/headless/v3/headless.esm.js');
  const {context, ...restOfConfiguration} = getSampleCommerceEngineConfiguration();

  const configuration = {
    context: {
      ...context,
      view: {
        url: `${context.view.url}/browse/promotions/ui-kit-testing`,
      },
    },
    ...restOfConfiguration,
  };

  await customElements.whenDefined('atomic-commerce-interface');
  const commerceInterface = document.querySelector('atomic-commerce-interface');
  await commerceInterface.initialize(configuration);
  await commerceInterface.executeFirstRequest();
</script>
```